docs/code/getting-started.md

```diff
@@ -486,57 +505,58 @@ Did you commit and push to GitHub for each step? Keep up these good habits and l
 
 Now it's time to create the rules that will allow your app to be built as a .deb package. Let's dive right in:
 
-1. Like CMake, elementary maintains a simple version of the "debian" folder that contains all the files we need for packaging. Let's grab a copy of that with `git`:
+1. elementary maintains a simple version of the "debian" folder that contains all the files we need for packaging. Let's grab a copy of that with `git`:
```

Comment:
Do we want to recommend an orphan branch, or is that out of scope here?

Reply:
If you're trying to groom this branch to release it "now" then probably not since the V2 worker isn't in production yet. But if you're planning to do that as a separate branch then yeah probably

Reply:
Oh I didn't realize that was still the case. I should go back and revert my splitting out for my AppCenter apps… :grimacing: